src/agent/block_store_services/block_store_fs.js

```diff
@@ -256,6 +260,86 @@ class BlockStoreFs extends BlockStoreBase {
         }
     }
 
+
+    /**
+     * @param {string[]} block_ids
+     * @param {string} storage_class
+     * @returns {Promise<{ moved_block_ids: string[] }>}
+     */
+    async _move_blocks_to_storage_class(block_ids, storage_class) {
+        if (storage_class === 'GLACIER') {
+            return this._move_blocks_to_glacier(block_ids);
+        }
```

Comment:
Just to avoid confusion, I would change to not call these block_store_fs methods with *glacier in their name, because while the storage class is called glacier, we are translating it to TMFS. So I think I would so something like this here, and if TMFS is not enabled we should probably continue and throw the error (unsupported storage class ) because we don't have a glacier tier to move to. WDYT?

```suggestion
        if (storage_class === 'GLACIER') {
            if (config.BLOCK_STORE_FS_TMFS_ENABLED) {
                return this._move_blocks_to_tmfs(block_ids);
            }
        }
```